### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by lukaw3d**
Fixes "invalid nonce" part of https://github.com/oasisprotocol/oasis-wallet-ext/issues/153

### CODE REVIEW

File: src/background/service/APIService.js [hunk 1]
```diff
@@ -755,7 +755,9 @@ class APIService {
                 const tw = oasis.staking.allowWrapper()
                 params.method = TRANSACTION_TYPE.StakingAllow
                 params.toAddress = oasis.staking.addressToBech32(await oasis.staking.addressFromRuntimeID(oasis.misc.fromHex(params.runtimeId)))
-                let result = await this.submitTxBody(params, tw,true,(data)=>this.depositToParatimeAccount(params,resolve,reject,data)).catch(err=>err)
+                let result = await this.submitTxBody(params, tw, true, (data) => {
```
Comment (buberdds) [inline src/background/service/APIService.js:758]: no need to use curly braces here

File: public/manifest.json [hunk 2]
```diff
@@ -37,7 +37,6 @@
     "scripts": ["common.js", "background.js"],
     "persistent": true
   },
-  "web_accessible_resources": [
-    "oasis-xu-frame.html"
-    ]
+  "web_accessible_resources": ["oasis-xu-frame.html"],
+  "externally_connectable": { "ids": [] }
```
Comment (buberdds) [inline public/manifest.json:41]: What's a benefit of adding this entry when empty `ids` array is the same as unspecified `externally_connectable` manifest key (as long as this theory is true)?

**Review by buberdds — APPROVED** [review]

File: public/manifest.json (hunk 2 above)
Comment (lukaw3d) [reply public/manifest.json:41]: Empty `externally_connectable` would do the same. But I think this is easier to understand intuitively, without looking at documentation :D 
and I can't add a comment

File: src/background/service/APIService.js (hunk 1 above)
Comment (lukaw3d) [reply src/background/service/APIService.js:758]: My preference is to have a curly for multiline

Comment (pro-wh) [reply src/background/service/APIService.js:758]: also nice to use braces for statements (even single statements imo) and no braces for expressions

**Review by pro-wh — APPROVED** [review]

**Pull request merged**